src/components/specimen/components/contentBlocks/specimenBlock/GeoReference.tsx

```diff
@@ -1,58 +1,57 @@
 /* Import Dependencies */
-import { Card } from 'react-bootstrap';
-import { MapContainer, TileLayer, Marker, Popup } from 'react-leaflet';
-import { Icon } from 'leaflet';
+import { Card, Row, Col } from 'react-bootstrap';
+import { isEmpty } from 'lodash';
 
 /* Import Store */
 import { useAppSelector } from "app/hooks";
 import { getSpecimen } from "redux/specimen/SpecimenSlice";
 
-/* Import Sources */
-import markerIconPng from 'leaflet/dist/images/marker-icon.png';
+/* Import Types */
+import { Dict } from 'app/Types';
 
 /* Import Icons */
 import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
 import { faGlobe } from '@fortawesome/free-solid-svg-icons';
 
+/* Import Components */
+import GeologicalMap from 'components/general/geologicalMap/GeologicalMap';
+
 
 const GeoReference = () => {
     /* Base variables */
     const specimen = useAppSelector(getSpecimen);
+    let georeference: Dict = {};
+
+    console.log(specimen.digitalSpecimen);
 
     /* Check if latitude and longitude are present */
```

Comment:
Doesn't completely align with the code. Code only checks if a georeference is present.

Reply:
Updated comment